cli/commands/run/server/agent.controller.js

```diff
@@ -13,8 +14,22 @@ module.exports = class AgentController {
   }
 
   async Initialize(call, callback) {
-    // no-op response
-    callback(null, { status: "SUCCESS" });
+    let initializeResponse = {}
+    let status = "SUCCESS";
+
+    if (this.initialize) {
+      try {
+        initializeResponse = await this.initialize();
+      } catch (e) {
+        console.log(e);
```

Comment:
can we format this error log the same as the others? it can help identify where exactly issues are